lib/jekyll-feed/generator.rb

```diff
@@ -27,11 +29,25 @@ def feed_path
       end
     end
 
+    # Path to JSON feed from config, or feed.json for default
+    def json_feed_path
+      if @site.config["feed"] && @site.config["feed"]["json_path"]
+        @site.config["feed"]["json_path"]
+      else
+        "feed.json"
+      end
+    end
+
     # Path to feed.xml template file
     def feed_source_path
       File.expand_path "./feed.xml", File.dirname(__FILE__)
     end
 
+    # Path to feed.json template file
+    def feed_json_source_path
+      File.expand_path "./feed.json", File.dirname(__FILE__)
```

Comment:
The feed_source_path function used that same format and I just changed the file path. Please note that I don't know any Ruby and I might be wrong.